assets/alertmanager/kube-rbac-proxy-secret.yaml

```diff
@@ -15,4 +15,9 @@ stringData:
       "rewrites":
         "byQueryParameter":
           "name": "namespace"
+      "static":
+        - "apiGroup": "monitoring.coreos.com"
+          "namespace": "{{ .Value }}"
+          "resource": "prometheusrules"
+          "resourceRequest": true
```

Comment:
we don't need static authorization for the /rules endpoint. The purpose is to be able to scrape metrics in the face of a network partition or a failed apiserver.